packages/web/tests/components/PopupModal/RemindMe.test.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import userEvent from '@testing-library/user-event';
+import fetchMock from 'fetch-mock';
+import { render, screen, waitFor } from '../../testUtils/testTools';
+import { RemindMeModal } from '../../../src/components/PopupModal/RemindModal';
+
+describe('Popup Modal', () => {
+  beforeEach(async () => {
+    jest.clearAllMocks();
+  });
+
+  it('renders correctly', async () => {
+    render(<RemindMeModal />);
+
+    const openModalButton = screen.getByText('Remind Me');
+    expect(openModalButton).toBeVisible();
+    userEvent.click(openModalButton);
+
+    await waitFor(() => {
+      expect(screen.queryByText('Confirm')).toBeVisible();
+    });
+
+    fetchMock.postOnce('/api/subscription/subscribe', { res: { ok: true } });
+
+    const confirmActionButton = screen.getByText('Confirm');
+    userEvent.click(confirmActionButton);
+
+    await waitFor(() => {
+      expect(screen.queryByText('Close')).toBeVisible();
+      expect(screen.queryByText('Success', { exact: false })).toBeVisible();
+    });
```

Comment:
Since this is duplicating a lot of the logic that's in the `PopupModal` tests, I would mock out the `PopupModal` component, and then get the `onConfirm` handler from that mocked component and just test that. Same for the other test in here

Example for fulfilled promise
```ts
await expect(PopupModalMock.mock.calls[0][0].onConfirm()).resolves.toBe(expect.anything());
```

Example for rejected promise
```ts
await expect(PopupModalMock.mock.calls[0][0].onConfirm()).rejects.toBe(expect.any(Error));
```